src/components/menu/menu.scss

```diff
@@ -21,16 +24,29 @@
   }
 }
 
+body[data-theme="dark"] .actmenu {
+  --bg1: rgba(44, 44, 44, 0.8);
+  --bg2: rgba(255, 255, 255, 0.08);
+
+  .menuopt img {
+    filter: hue-rotate(180deg) invert(1);
+  }
+  .menuopt:nth-child(11) img,
+  .menuopt:nth-child(12) img {
+    filter: none;
```

Comment:
?

Reply:
these are to prevent terminal icon and about icon from hue-rotate and invert effect

Reply:
which can temporary work but will be issue to resolve later